dts/arm/renesas/smartbond/da1469x.dtsi

```diff
@@ -332,6 +332,13 @@
 			interrupts = <9 0>;
 		};
 
+		display: lcdc@30030000 {
```

Comment:
Instead of defining the same IP block twice with different compatibles, you should override the compatible of the device in the overlay file, something like so:

da1469x.dtsi:
```
lcdc: lcdc@30030000 {
    	compatible = "renesas,smartbond-display";
	reg = <0x30030000 0x18C>;
	interrupts = <32 0>;
	status = "disabled";
};
```
da1469x_dk_pro_mipi_dbi.overlay
```
&lcdc {
    status = "okay";
    compatible = "renesas,smartbond-mipi-dbi";
};
```

Reply:
Thanks for the hint. I followed your suggestion but I am now unable to build. In specific, when I try to build for the display compatible via
`west build -p always -b da1469x_dk_pro samples/drivers/display/ -- -DDTC_OVERLAY_FILE="da1469x_dk_pro_lcdc.overlay"` I am getting errors on missing properties of the MIPI DBI compatible. The same goes if I build for the MIPI DBI compatible. Could you please check?
<img width="953" alt="display_build_error" src="https://github.com/zephyrproject-rtos/zephyr/assets/13284615/bdedc651-e945-4b9e-bf1d-9244d975ddc4">
<img width="956" alt="mipi_dbi_build_error" src="https://github.com/zephyrproject-rtos/zephyr/assets/13284615/a8ed67b0-8216-49f9-b585-6123fcd3a2b5">

Reply:
Forget my previous comment; I messed up the compatible properties between the display and MIPI DBI overlays :/

Reply:
No worries! Thanks for getting this changed :)